pages/[token].js

```diff
@@ -51,7 +58,7 @@ const Share = ({ cert }) => {
                                     <Button className="w-100"
                                             icon="arrow"
                                             variant="primary"
-                                            href={`https://www.linkedin.com/sharing/share-offsite/?url=${path}`}
+                                            href={`https://www.linkedin.com/sharing/share-offsite/?url=${generateShareUrl()}`}
```

Comment:
This will return something like: https://www.linkedin.com/sharing/share-offsite/?url=preview?cacheBuster=hash.

I think the best approach is to include this as a querystring of the LinkedIn site

Reply:
I tried to add cacheBuster as a query string but it did not seem to fix the issue:

https://www.linkedin.com/sharing/share-offsite/?url=certificate.4geeks.com/3ee4ddacfda402cc7a75bc0301813bdfde87cb68&cacheBuster=1724710001714

![image](https://github.com/user-attachments/assets/0cd2d15e-f429-4515-8cc4-c3ba7a4ea484)

It looks like LinkedIn just ignores the cacheBuster query param and only focuses on the url param, resulting in an 'invalid URL' preview.